### DISCUSSION TITLE
When a new release is planned

### DISCUSSION THREAD

**Discussion by mavnino** [TWiStErRob/net.twisterrob.inventory#470] [Questions]
Hi

I see that the problem with search in Cyrillic is already solved in main stream
When a new release with this fix is planned?

**Comment by TWiStErRob** [comment]
It's not fully solved yet until I merge https://github.com/TWiStErRob/net.twisterrob.inventory/pull/357. I was thinking of pushing a release after that's done.

My life has been quite busy this spring and summer, so didn't get around to work on this. I might find some time during the dark winter evenings.

**Reply by mavnino** [reply]
Hi!

Looks like the project is fully stopped? :(

**Reply by TWiStErRob** [reply]
It's in maintenance, not actively working on it. Life is going fast 🫣.
